packages/recommend-vdom/src/types/ViewProps.ts

```diff
@@ -6,7 +6,11 @@ export type ViewProps<
   TClassNames extends Record<string, string>
 > = {
   classNames: TClassNames;
-  itemComponent({ item: TItem }): JSX.Element;
+  itemComponent({
+    item: TItem,
+    createElement: Pragma,
+    Fragment: PragmaFrag,
+  }): JSX.Element;
```

Comment:
And the missing import.

```suggestion
  itemComponent(props: { item: TItem } & Renderer): JSX.Element;
```